recipes/flask-multipass/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "flask-multipass" %}
+{% set version = "0.3.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/Flask-Multipass-{{ version }}.tar.gz
+  sha256: 536adcabb4db117556a8c92db71fb4df647d8ee3292dc5c39f444194c8f512b6
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [win or py < 36]
```

Comment:
Is there a reason that this package should be skipped on windows? Otherwise it seems like this could be a `noarch: python` with the python version constrained under `host` and `run`.

Reply:
The `python-ldap` package is not available for Windows.

Reply:
(fwiw, that's one of the reason why it's an `[ldap]` extra in the original package)

Reply:
General conda-forge policy is to make this `noarch: python` since the package is noarch even if conda-forge is missing a dependency. See the 3rd note under:
https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Reply:
I tried that this morning, but then the windows CI went red 😢

Reply:
`staged-recipe` admins will merge a PR with red CI flags if the reason it's failing is understood to be a missing dependency.